index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="toDoList">
+<head>
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="CSS/styles.css">
+  <script src="bower_components/jquery/dist/jquery.min.js"></script>
+  <script src="bower_components/angular/angular.min.js"></script>
+  <script src="js/toDoController.js"></script>
+  <meta charset="UTF-8">
+  <title>toDo Lister</title>
+</head>
+
+<body ng-controller="toDoController">
+
+  <div class="container" >
+
+    <h3 class="list-group-item task">ToDo Lister</h3>
+    <h4 class="list-group-item count">Total number of tasks: {{totalTasks()}} </h4>
+    <ul class='list-group-item'>
+      <li  ng-repeat="taskItem in allTasks | filter: done">
+        <input id="completed" type="checkbox" ng-change="setChecked($index)" ng-model="done"></input>
```

Comment:
always prefer passing an object instance instead of an arbitrary identifier like `$index`.  You are relying on the view index being the same as the index in the source collection which is very dangerous.